WordPress/Classes/ViewRelated/People/PeopleViewController.swift

```diff
@@ -37,7 +37,7 @@ public class PeopleViewController: UITableViewController, NSFetchedResultsContro
     }
     
     override public func tableView(tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
-        return CGFloat.min
+        return isViewHorizontallyCompact() ? CGFloat.min : 0
```

Comment:
`isViewHorizontallyCompact` is defined in the stats pod so its a bit weird to call it from here.  If the `UIViewController+SizeClass` category belonged in the shared pod it would be more proper. 
I'm fine leaving the code as is if we can make it a shared category (future pr is okay), other wise I think it would be better to check the traitCollection's size class.